### ISSUE TITLE
java.net.SocketTimeoutException after upgrading Chromium

### ISSUE THREAD

**Issue by Outrovurt** [clj-commons/etaoin#373]
All my etaoin tests, which are written to run on just a chrome driver, were working perfectly fine until I recently upgraded my Archlinux system which brought my Chromium up from version `95.0.4638.69` to `96.0.4664.93`, and now when I do something like the following:

```clojure
(def dr (chrome))
```

the chrome browser window opens, it displays "data:," in the URL, and pretty much goes on loading indefinitely until it throws a `java.net.SocketTimeoutException`.

I tried to rollback my Chromium install but then ran into dependency issues so was unable to verify that that was the cause, however I'm 99% certain that it is.

Using the firefox driver works perfectly fine, however I had to rewrite some tests as firefox handles the `value` attribute differently from `chrome`. This was not an issue.

The strange thing is that if I run the chrome driver as headless, it works fine.

My question is, has anyone come across a similar issue to this in the past, and if so is there any way of dealing with it? I appreciate this is not an issue with `etaoin`, I was just curious if there is something straightforward I can do to address it.

Along those lines, is there a general process for diagnosing such issues? I've tried passing different options as the second argument to `boot-driver`, however I haven't as yet been successful in getting the none-headless chrome driver to work.

**Comment by lread** [comment]
Hi @Outrovurt, thanks for raising this issue.

I'm going to make the bold assumption that the problem was with that version of chromedriver and things are now working well for you.

But... if my bold assumption is wrong, please do re-open this issue and we'll diagnose together.

As for diagnosis, the first thing I'd try would be enabling the logs and maybe even the driver log level.

```Clojure
(require '[etaoin.api :as e])

(def driver (e/chrome {:log-stdout "chromedriver-out.log"
                       :log-stderr "chromedriver-err.log"
                       :driver-log-level "DEBUG"}))
```
And then I'd take a peek in those logs to see if they held any clues.

See [driver options](https://github.com/clj-commons/etaoin/blob/master/doc/01-user-guide.adoc#driver-options) for details.

**Comment by Outrovurt** [comment]
Hi Lee, 

Your bold assumption turns out to be correct after all, but I've only just discovered that myself. As an alternative to using chromedriver, I had switched out the `:chrome` argument to `with-driver` for `:firefox` and have been working with that all this time. Today I switched back to `:chrome` and it being a few versions on from when it was breaking, it now seems to be working again.

Thanks for the tip on capturing the logs, but I'm relieved (as I'm sure you are) that we didn't have to go down that route!
